src/testFixtures/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/resource/model/induction/FutureWorkInterestsResponseAssert.kt

```diff
@@ -0,0 +1,128 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.resource.model.induction
```

Comment:
A handful of new assertJ assertion classes for the child properties of `InductionResponse` - these new assertion classes are used within `InductionResponseAssert` below